README.md

```diff
@@ -10,38 +10,35 @@ To run the API you will need appropriate AWS credentials. See the [Wiki](https:/
 
 ### pycurl
 
-pycurl needs to be installed separately, with some specific compiler flags and steps. You'll need to follow these steps
-in any repo where you wish to run celery locally. The following steps have been adapted from https://gist.github.com/vidakDK/de86d751751b355ed3b26d69ecdbdb99
```

Comment:
I'd be tempted to keep this paragraph, I think useful context as this may change a bit more and the gist will get updated over time potentially